packages/playground/src/weblets/full_vm.vue

```diff
@@ -176,6 +181,11 @@ const images: VmImage[] = [
     flist: "https://hub.grid.tf/tf-official-vms/ubuntu-18.04-lts.flist",
     entryPoint: "/init.sh",
   },
+  {
+    name: "Arch",
+    flist: "https://hub.grid.tf/petep.3bot/arch_mycelium.flist",
+    entryPoint: "/init.sh",
+  },
```

Comment:
it shouldn't be in the fullvm

Reply:
Why?

Reply:
cause it doesn't contain a kernel, it just created the file system using dockerfile

Reply:
Removed.